setup.py

```diff
@@ -1,31 +1,44 @@
 from setuptools import setup, find_packages
-import os.path
-
-def read(path):
-    return open(
-        os.path.join(os.path.dirname(__file__), *path.split('/'))).read()
-
 
 setup(
     name="zc.set",
-    version="0.1dev",
+    version="0.2",
     license='ZPL 2.1',
     long_description='\n\n'.join([
-        read('CHANGES.txt'),
-        read('src/zc/set/README.txt'),
+        open('CHANGES.txt').read(),
+        open('src/zc/set/README.txt').read(),
         ]),
     packages=find_packages('src'),
-    package_dir={'':'src'},
+    package_dir={'': 'src'},
     namespace_packages=['zc'],
     include_package_data=True,
     install_requires=[
         'setuptools',
-        'ZODB3',
-        'zope.app.folder',
+        'persistent',
     ],
     extras_require=dict(
         test=[
-            'zope.testing',
+            'ZODB[test]',
+            'zope.app.folder',
             ]),
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Zope Public License',
+        'Natural Language :: English',
+        'Operating System :: POSIX',
+        'Operating System :: Microsoft :: Windows',
```

Comment:
IIRC there's a classifier for "os independent".

Reply:
[`Operating System :: OS Independent`](https://pypi.org/search/?c=Operating+System+%3A%3A+OS+Independent)